packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 5.0.6 | [PR#2377](https://github.com/bbc/psammead/pull/2377) Spread extra props to the component |
```

Comment:
Should this be a breaking change as well, considering the others are?

Reply:
The patch should be fine since we are not removing props as in the others.